js/ui/control/attribution_control.js

```diff
@@ -67,11 +67,11 @@ AttributionControl.prototype = util.inherit(Control, {
             }
             return true;
         });
-
         this._container.innerHTML = attributions.join(' | ');
     },
 
     _updateEditLink: function() {
+        if (!this._editLink) this._editLink = this._container.querySelector('.mapbox-improve-map');
```

Comment:
When, if ever, does the "edit link" element get replaced in the DOM? Will we update `this._editLink` when the element is replaced?